tests/test_env.rs

```diff
@@ -13,9 +16,66 @@ const COMMUNITY_FACTORY_PREFIX: &str = "community";
 const NEAR_SOCIAL: &AccountIdRef = AccountIdRef::new_or_panic("social.near");
 const _TEST_NEAR_SOCIAL: &AccountIdRef = AccountIdRef::new_or_panic("v1.social08.testnet");
 const TEST_SEED: &str = "testificate";
-const DEVHUB_CONTRACT_PATH: &str = "./target/near/devhub.wasm";
-const COMMUNITY_FACTORY_CONTRACT_PATH: &str =
-    "./community-factory/target/near/devhub_community_factory.wasm";
+
+const WASM_FOR_TESTS_DIRECTORY: &str = "./target/devhub-tests-contracts";
+
+pub static DEVHUB_CONTRACT_WASM: LazyLock<Vec<u8>> = LazyLock::new(|| {
+    let expected_path = std::path::PathBuf::from_str(WASM_FOR_TESTS_DIRECTORY)
+        .expect("path from str")
+        .join("devhub.wasm");
+    if !std::fs::exists(&expected_path).expect("std::fs::exists call") {
+        cargo_near_build::build(cargo_near_build::BuildOpts {
+            out_dir: Some(
+                cargo_near_build::camino::Utf8PathBuf::from_str(WASM_FOR_TESTS_DIRECTORY)
+                    .expect("camino PathBuf from str"),
+            ),
+            ..Default::default()
+        })
+        .expect("building `devhub` contract for tests");
+    }
```

Comment:
I'd build it regardless of whether the file exists to ensure that the test runs against the current version of the code (see my comment in the README). My hope is that the incremental build for no-changes will be instant.

Reply:
hopefully handled by https://github.com/NEAR-DevHub/neardevhub-contract/pull/149/commits/6d44ceb17f69abb472fab5ba2b5eb89598c61d12